server/controllers/ArticleController.js

```diff
@@ -135,11 +138,11 @@ class ArticleController {
       title, description, body,
     } = req.body.article;
     const authorId = req.body.decoded.id;
-    const updateField = [];
+    const fieldsArr = [];
 
-    if (title && title.trim()) updateField.push('title');
-    if (description && description.trim()) updateField.push('description');
-    if (body && body.trim()) updateField.push('body');
+    if (title) fieldsArr.push('title');
```

Comment:
The 'updateField' specified as the variable name is preferable over the 'fieldsArr' as specified in previous comments on the CRUD PR. The current changes should have been accepted upon rebase to the create article branch to avoid reverting back to old changes

Reply:
use git reflog to go back to how your branch was before rebase and then rebase again to accept the right changes